react/wbooks/src/app/screens/Home/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+import { getBooks } from '../../../services/Book/service';
+import { ACCESS_TOKEN } from '../../../constants';
+
+import Home from './layout';
+
+class HomeContainer extends Component {
+  state = { books: [] };
+
+  async componentDidMount() {
+    const books = await getBooks(localStorage.getItem(ACCESS_TOKEN));
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({ books });
```

Comment:
extract this to a function, so you are not doing async componentDidMount

Reply:
that is bad?

Reply:
async componentDidMount is for React Native only, is really weird in react, you are making a life hook wait to a request, https://stackoverflow.com/questions/47970276/is-using-async-componentdidmount-good read the answers and is up to you.